doc/_static/mpl.css

```diff
@@ -1118,3 +1118,28 @@ div.viewcode-block:target {
 .sidebar-announcement p {
     margin: 0.4em 0.4em 0.6em 0.4em;
 }
+
+/* new main nav */
+nav.main-nav{
+    background-color: #002b47;
+    font-family: Calibri, 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
```

Comment:
I don't see that we use Calibri anywhere else.  Is there a reason for this choice?  Just like colors, I find the diversity of fonts on the front-page impressive enough already without adding more.

Reply:
Calibri is the font used by the logo text. Which means my screenshot above is actually using Calibri and the screenshot in the original post is falling back to some other font.

Reply:
Sure, but then lets make a lot more of the header text `Calibri` if thats what we want.  I'm not too fussed if the logo (or plots) is a different font because it is quite distinctive, but I'd prefer we minimize the html fonts.

Reply:
Calibri has a proprietary license. While I think that's not an issue as long as we just reference it in CSS but don't distribute it, it might be nice to use a font with an open license for an open source project. Also, it's usually only available on Windows, so many users will fall back to something else anyway.

Reply:
Yup, Calibri is the font used in the logo and it is a licensed font (though many people have it installed on their computers via microsoft products). The `font family` property will load Calibri if visitors already have it already installed on their computers. Otherwise, browsers will go down the list until it finds a font that matches. There are open source alternatives that we could load with the webpage, but this would be a bigger projects thats best handled in a different conversation about branding (and maybe sticker budgets)

Helvetica is the only font currently used as browser rendered text on the site–Here's the font stack:
https://github.com/matplotlib/matplotlib/blob/349e2e9e8a33c5a45d2935aae86561bdea10e55b/doc/_static/mpl.css#L6-L7)
 
I usually design slightly different treatments for header navigation text, since it serves a different purpose on a webpage from text or headers in the main content (a nav helps site visitors find a page and text is for absorbing content). I did try doing something with Helvetica, but it only contributed to the page looking like a wall of text.